.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "printWidth": 100,
+    "useTabs": false,
+    "tabWidth": 2,
+    "semi": false,
```

Comment:
Not sure if we should disable semicolons and trailing commas here, but i believe this is something that'd require input from others.

Reply:
@chocoelho Do you have any thoughts on it?